test/test_sparse.cc

```diff
@@ -162,7 +162,7 @@ class TestLSKGES : public ::testing::Test
                 layout, blas::Op::NoTrans, blas::Op::NoTrans,
                 d, n, m,
                 1.0, sas, 0, 0, a, lda,
-                0.0, a_hat, ldahat
+                0.0, a_hat, ldahat, 1
```

Comment:
Related to my comment in the new ``RandBLAS/sparse.hh`` file: ``lskges`` used to have a default value of 4 for this parameter. I'd like to go back and continue to use a default value for this parameter so that ``lskges`` and ``lskge3`` can have the same signature.

Reply:
OK, agree with you about them having the same signature.